build.gradle

```diff
@@ -30,15 +30,15 @@ buildscript {
 }
 
 plugins {
-  id 'com.diffplug.spotless' version '5.10.2'
-  id 'com.github.ben-manes.versions' version '0.36.0'
+  id 'com.diffplug.spotless' version '5.12.4'
+  id 'com.github.ben-manes.versions' version '0.38.0'
   id 'idea'
   id 'java-library'
   id 'org.owasp.dependencycheck' version '6.1.1'
```

Comment:
just curious. Why not updating other plugins?

Reply:
Maybe I just missed it when looking at the updates. Are you thinking of the dependency check? The shadow upgrade requires Gradle 7, so that has to wait.

Reply:
The latest version of `org.owasp.dependencycheck` is 6.1.6. Maybe we can update it also?

Reply:
Done.